src/Redux/store/authSlice.js

```diff
@@ -0,0 +1,29 @@
+import { createSlice } from "@reduxjs/toolkit";
+import {
+  getLocalStorage,
+  removeLocalStorage,
+  setLocalStorage,
+} from "../../helper/localStorage";
+
+const authSlice = createSlice({
+  name: "auth",
+  initialState: {
+    token: JSON.parse(getLocalStorage("user")) || null, // Get the token from local storage
```

Comment:
This is wrong to store the user in a token store only the token in the key "token" or update the name.